src/index.js

```diff
@@ -41,16 +41,19 @@ class Service {
     return Proto.extend(obj, this);
   }
 
-  init (opts = {}) {
+  init () {
```

Comment:
I would leave this in here for consistency as I'm planning to add the same signature in other adapters and this may eventually expand.